Activity.js

```diff
@@ -0,0 +1,19 @@
+class Activity {
+  constructor(category, description, minutes, seconds) {
+    this.id = Date.now();
+    this.category = category;
+    this.description = description;
+    this.minutes = minutes;
+    this.seconds = seconds;
+    this.completed = false;
+  }
+    startTimer(){
+
+    }
```

Comment:
Also let's add a line of whitespace between the closing curly brace and the function declaration.